js-exercises/array-cube-root-to-json/arrayCubeRootToJson.js

```diff
@@ -1,3 +1,34 @@
-const arrayCubeRootToJson = arr => {};
+const isNumberValid = num => {
+  if (typeof num === 'number' || (typeof num === 'string' && !isNaN(Number(num)))) {
```

Comment:
It should check for null and boolean also.
typeof not necessary. isNaN function does that work. No need to pass Number(num) to isNaN.
You can do something like if (num === null || num === false || isNaN(num)) throw new Error();